openslides/users/auth.py

```diff
@@ -31,11 +30,12 @@ def get_group_permissions(self, user_obj, obj=None):
             if user_obj.is_superuser:
                 perms = Permission.objects.all()
             else:
-                user_groups_field = get_user_model()._meta.get_field('groups')
-                user_groups_query = 'group__%s' % user_groups_field.related_query_name()
-                # The next two lines are the customization.
-                query = Q(**{user_groups_query: user_obj}) | Q(group__pk=2)
-                perms = Permission.objects.filter(query)
+                if len(list(user_obj.groups.all())) == 0:  # user is in no group
```

Comment:
Please change to `user_obj.groups.all().count() == 0:` which I successfully tested with @normanjaeckel